webapp/public/index.html

```diff
@@ -3,6 +3,7 @@
 <head>
     <!-- Global site tag (gtag.js) - Google Analytics -->
     <script async src="https://www.googletagmanager.com/gtag/js?id=UA-122398180-1"></script>
+    <script src="https://apis.google.com/js/api.js"></script>
```

Comment:
did you ever check if you needed this to not be async? Can you verify that we can't have async there cause it slows down initial page load time